docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+  With cloud sync enabled for a component, data captured locally to your smart machine is automatically deleted after a successful sync.
+  Data synced between your smart machine and the Viam app is encrypted in transit (over the wire) and when stored in the cloud (at rest).
+
+Data capture and data sync are frequently used together, and are both enabled by default when you add the data management service to your smart machine.
+However, if you want to manage your smart machine's captured data yourself, you can enable data capture but disable data sync.
+
+To capture data from your smart machine and sync to the Viam app, add the data management service and configure data capture for at least one component.
+
+### Add the data management service
+
+First, add the data management service to your smart machine to be able capture and sync data:
+
+1. On your robot's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Services** tab.
+1. Click the **Create service** button at the bottom of the page, and select **Data Management**.
+1. Give the service a name, like `viam-data-manager`, then click **Create Service**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data to the <file>~/.viam/capture</file> directory, and syncs captured data files to the Viam app every 6 seconds (`0.1` minutes in the configuration).
+   Leave the default settings as they are, and click **Save Config** at the bottom of the screen to save your changes.
+
+   {{< imgproc src="/tutorials/data-management/data-management-conf.png" alt="The data management service configuration pane with default settings shown for both capturing and syncing" resize="600x" >}}
+
+For more information, see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+
+### Configure data capture for a component
+
+Once you have added the data management service, you can configure data capture for specific components on your smart machine.
+For this tutorial, you will configure data capture for images from a [camera](/components/camera/) component, but other data types such as sensor data or SLAM map data from other types of [components](/components/) can be captured as well.
+
+To enable image data capture for a camera component:
+
+1. On your robot's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
+1. If you have not already, add a [camera component](/components/camera/), such as a [webcam](/components/camera/webcam/), to your smart machine.
+   If you are using a Viam Rover, the camera is pre-configured, and named `cam`.
+
+1. In your camera component configuration pane, find the **Data capture configuration** section, and click the **Add method** button to enable data capture for this camera.
```

Comment:
```suggestion
1. In the camera component configuration pane from your [configured camera](#prerequisites), find the **Data capture configuration** section, and click the **Add method** button to enable data capture for this camera.
```

Reply:
Ah, I see now. You're right, this is better!